reference/pipeline/templates/index.md

```diff
@@ -80,4 +81,6 @@ config:
   "description": "",
   "stages": []
 }
-```
\ No newline at end of file
+```
+
+In `.template` section, `http://` and `file://` prefixes are also supported.
```

Comment:
I believe this functionality was removed in favor of template sources always resolving to artifacts: https://github.com/spinnaker/orca/pull/2667

Reply:
Thanks for the reference, I removed the comment.